python/ecs/ecs-service-with-task-placement/app.py

```diff
@@ -6,7 +6,7 @@
 )
 
 app = App()
-stack = Stack(app, "aws-ecs-integ-ecs")
```

Comment:
aws-ecs-integ-ecs is concatenated with the construct id in capacity provider to be used as the resource name... the resource name however can't start with 'aws'